example/bwmf/bwmf_task.go

```diff
@@ -397,7 +394,25 @@ func (t *bwmfTask) GetDShard(ctx context.Context, request *pb.Request) (*pb.Resp
 }
 
 func (t *bwmfTask) MetaReady(ctx context.Context, fromID uint64, linkType, meta string) {
-	t.metaReady <- &event{ctx: ctx, fromID: fromID}
+	t.logger.Printf("MetaReady, task %d, from: %d, linkType: %s, meta: %s", t.taskID, fromID, linkType, meta)
+	switch linkType {
+	case "Master":
+		go t.receiveUpdateDone(ctx, fromID)
+	case "Neighbors":
```

Comment:
Neighbors should be changed to "workers" because Neighbors is a very common name.